config/default.json

```diff
@@ -66,7 +66,8 @@
     "springSale30PercentOff",
     "showMoneyBackGuarantee",
     "multiyearSubscriptions",
-    "signupDomainStrikethruPrice"
+    "signupDomainStrikethruPrice",
+    "mothersDay2018Discount_20180411"
```

Comment:
Are you sure that date is required here? It looks like date should be present in `overrideABTests` only